metrics/web/test/auth/presentation/widgets/auth_form_test.dart

```diff
@@ -54,64 +51,71 @@ void main() {
     );
 
     testWidgets(
-        "email input shows an error message if a value is not a valid email",
-        (WidgetTester tester) async {
-      await tester.pumpWidget(_AuthFormTestbed(authNotifier: authNotifier));
-      await tester.enterText(emailInputFinder, 'notAnEmail');
+      "email input shows an error message if a value is not a valid email",
```

Comment:
```suggestion
      "email input shows an error message if a value is not a valid email on submit",
```
should we add "on submit" everywhere as that will make description more precise and highlight what it does?